MvvmCross/tvOS/tvOS/Views/Presenters/Attributes/MvxDetailSplitViewPresentationAttribute.cs

```diff
@@ -0,0 +1,9 @@
+﻿using MvvmCross.Core.Views;
+namespace MvvmCross.tvOS.Views.Presenters.Attributes
+{
+    public class MvxDetailSplitViewPresentationAttribute : MvxBasePresentationAttribute
```

Comment:
Please merge this into one, like https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross-Forms/MvvmCross.Forms/Views/Attributes/MvxMasterDetailPagePresentationAttribute.cs#L24